docs/wiki/PipelinesDesign.md

```diff
@@ -218,14 +218,13 @@ The [pipeline configuration file](#pipeline-variables) can be found at `.github/
 
 We use several composite actions to perform various tasks shared by our module workflows:
 
-- **validateModuleGeneral** <p>
-  This action perform all [static tests](#static-module-validation) for a module using Pester.
-- **validateModuleApis** <p>
-  This action performs further [static tests](#static-module-validation), focused on API version tests for a module to ensure we notice if a version becomes stale. Each resource's API version is compared with those currently available on Azure. Accepted are both the latest 5 versions (including preview versions) as well as the latest 5 non-preview versions.
-- **validateModuleDeploy:** <p>
-  This action performs the [simulated deployment](#simulated-deployment-validation) using a provided parameter file.
-- **deployModule:** <p>
-  This action performs an [actual deployment](#test-deploy) to Azure using a provided parameter file. Once a deployment ran it [removes](#removal) the resource
+- **validateModulePester** <p>
+  This action perform all [static tests](#static-module-validation) for a module using Pester. It also performs further [static tests](#static-module-validation), focused on API version tests for a module to ensure we notice if a version becomes stale. Each resource's API version is compared with those currently available on Azure. Accepted are both the latest 5 versions (including preview versions) as well as the latest 5 non-preview versions.
```

Comment:
```suggestion
  This action performs [static tests](#static-module-validation) for a module using Pester, including API versions focused tests to avoid those become stale overtime.
```

Reply:
I'd keep this short as we're already linking to the dedicated section